config/initializers/global_functions.rb

```diff
@@ -10,10 +10,18 @@ def production?
   Rails.env.production?
 end
 
+def staging?
+  Rails.env.staging?
+end
+
 def development_or_test?
   development? || test?
 end
 
 def development_or_production?
```

Comment:
I would like to test MailChimp subscription and Postmark on both, but not on development and testing. is this a normal procedure?